src/SUMMARY.md

```diff
@@ -7,7 +7,9 @@
   - [Engine](./geo-engine/engine.md)
   - [Workflows](./geo-engine/workflows.md)
 
-- [Operators](./operators.md)
+- [Operators](./operators/intro.md)
```

Comment:
Could you say 1-2 sentences as an intro to operators?

Reply:
79b36d69847bd722e348825fe0903e7cb05ee27d